packages/svelte-docgen/src/parser/props.test.ts

```diff
@@ -98,14 +98,7 @@ describe("props", () => {
 			          "kind": "undefined",
 			        },
 			        {
-			          "kind": "literal",
-			          "subkind": "boolean",
-			          "value": false,
-			        },
-			        {
-			          "kind": "literal",
-			          "subkind": "boolean",
-			          "value": true,
+			          "kind": "boolean",
```

Comment:
Huh, looks innocent. But we've overlooked this before - #40

Reply:
When `boolean` is part of a union in TypeScript, it is internally expanded to `true | false`. For example, `a?: boolean` expands to `true | false | undefined`. To prevent inconsistency, we likely need additional logic to combine `... | true | false | ...` into `... | boolean | ...` Such logic has been incorporated into my initial implementation for Storybook :)

Reply:
- #71